### ISSUE TITLE
elasticsearch depends on spatial test jar

### ISSUE THREAD

**Issue by rmuir** [elastic/elasticsearch#14328]
```
[INFO] +- org.apache.lucene:lucene-spatial:jar:5.4.0-snapshot-1708254:compile
[INFO] |  +- org.apache.lucene:lucene-spatial3d:jar:5.4.0-snapshot-1708254:compile
[INFO] |  \- com.spatial4j:spatial4j:jar:tests:0.5:compile
```

and make a .zip distribution and you see this tests jar in lib/

it should not be there...

**Comment by s1monw** [comment]
I think this is a lucene issue see the difference between 5.x and 5.3
`http://svn.apache.org/repos/asf/lucene/dev/branches/branch_5x/lucene/spatial/ivy.xml` vs. `http://svn.apache.org/repos/asf/lucene/dev/branches/lucene_solr_5_3/lucene/spatial/ivy.xml` I will open a lucene issue I think it's caused by this revision: http://svn.apache.org/viewvc?view=revision&revision=1704760 / https://issues.apache.org/jira/browse/LUCENE-6810

**Comment by s1monw** [comment]
I opened this https://issues.apache.org/jira/browse/LUCENE-6860

**Comment by clintongormley** [comment]
Fixed in Lucene 5.4
